src/components/PhotoGallery/PhotoGallery.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import axios from "axios";
+import Gallery from "react-photo-gallery";
+import { shuffleArray } from "utility/Helpers";
+
+const PhotoGallery = () => {
+  const [images, setImages] = React.useState<any[]>([]);
+
+  async function fetchImages() {
+    let response = await axios.get(
+      "https://google-photos-album-demo2.glitch.me/SkVei5N56poqTh8g8"
+    );
+    console.log(response.data);
+    if (response.data && response.data.length > 0) {
+      const shuffledResponseData = shuffleArray(response.data);
+      setImages(
+        shuffledResponseData.map((url: string) => ({
+          src: url,
+          width: 4,
+          height: 3,
```

Comment:
I've set a universal aspect ratio of 4:3 for all images - so we need to keep in mind to only upload landscape pics to our shared gallery album on Google Photos.